hack/verify-spelling.sh

```diff
@@ -45,7 +45,7 @@ cd "${ROOT}"
 RES=0
 echo "Checking spelling..."
 ERROR_LOG="${TMP_DIR}/errors.log"
-git ls-files | grep -v vendor | xargs misspell > "${ERROR_LOG}"
+git ls-files | grep -v vendor | xargs misspell -i $(grep -v '#' hack/.spelling_ignorewords) > "${ERROR_LOG}"
```

Comment:
I don't think this change belongs there

Reply:
Do we need to open another PR for it?

Reply:
Possibly, but I don't get why do we need this change in this PR?

Reply:
This is to fix a CI failure, my name (cyclinder) did not pass the misspell check, so I made these changes, but I am not sure if I need to open a new PR for it, I put it on this pr for now.  I can open another PR if it can be confirmed.

Reply:
thing is, usernames should not be spell-checked in the first place :\ I'll have a look.

Reply:
ok that's the failure: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/enhancements/4624/pull-enhancements-verify/1788125378728955904 - it strongly believe it should be a separate PR. Pending conversation, I think `kep.yaml` should not be spell-checked in the first place.

Reply:
I think this expands the scope a bit, checking kep.yaml is ok, but it should not include the author name.

Reply:
It's fair also to exclude usernames from spell-checking `kep.yaml`. Even in this case it should be a separate PR though.

Reply:
okay, maybe it's hard to exclude usernames from spell-checking `kep.yaml` for misspell :(

Reply:
This would be fixed in https://github.com/kubernetes/enhancements/pull/4720.